css/styles.css

```diff
@@ -14,19 +123,21 @@
 .background {
   margin: 0;
   height: 600px;
-  background-image: url("../background.png");
+  background-image: url("../imgs/background.png");
   background-position: center;
   background-repeat: no-repeat;
   background-size: cover;
 }
 
 .list-format {
   list-style-type: none;
-  display: flex;
-  justify-content: space-around;
   margin: auto;
   padding: 0;
-  width: 30%;
+}
+
+.list-grid {
+  display: grid;
+  grid-template: auto / repeat(9, 1fr);
 }
```

Comment:
Good job so far, However, Please ensure that your list of technologies design matches the given design as part of the requirement that says: `You must stick to the design as much as possible (e.g., font, colors, images, text, margins)` using this figma [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&mode=design&t=rPm74ST0zW6b1ScP-0) matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently.  Please ensure that you pay close attention the design  :slightly_smiling_face: 

The example below shows the difference between yours and the accepted one: :arrow_down: 

 <table>
    <tr>
      <td>Yours </td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![F Tlist](https://github.com/flov3rh/portfolio/assets/43528594/dcb0fe71-de5c-4fbf-893e-921855525f6a)

</td>
      <td>

![F tlist](https://github.com/flov3rh/portfolio/assets/43528594/c81b8920-5d10-42e6-8f04-23c53cd40e7e)

</td>
    </tr>
  </table>